src/test/java/com/Query/query/Controller/AlunoControllerTest.java

```diff
@@ -19,7 +19,7 @@
 	private MockMvc mockMvc;
 
 	@Test
-	private void retornaAlunoStatus200paraUmParametro() throws Exception {
+	public void retornaAlunoStatus200paraUmParametro() throws Exception {
 
 		URI uri = new URI("/aluno/1");
 

```

Comment:
Adicionar mais "andExpect" junto com org.hamcrest.CoreMatchers.is para verificar se os valores da entidade salvos no banco batem com o retornado pelo Controller. Aplicar essa alteração em todas as classes de testes.

Reply:
feito